single-page-app/verify.sh

```diff
@@ -217,12 +221,12 @@ test_auth () {
     echo "$RESPONSE" | grep "set-cookie: BearerToken=deleted"
 }
 
-get_js () {
-    _curl -k "https://localhost:${PORT_PROXY}" \
-        | grep "assets/index" \
-        | grep -oP '<script type="module" crossorigin src="/assets/[^"]+"></script>' \
-        | grep -oP '/assets/[^"]+' \
-        | sed 's/\/assets\///;s/".*//'
+get_js() {
+    _curl -k "https://localhost:${PORT_PROXY}" |
```

Comment:
i dont overly like most of the format changes but in this case i **strongly** disagree - mho is that code conjunctions _should_ be at start of line - i think it makes the code an order of magnitude easier to read